### PR TITLE
Moving mon, controller and seed to bond

### PR DESCRIPTION
**Pull request by JohnGarbutt**
Now we have the 25GbE cards installed, lets use them.

### CODE REVIEW

File: etc/kayobe/inventory/group_vars/controllers/network-interfaces [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+# Note: moved from breno3 1GbE interface
+br_net_interface: brbond0
 br_net_bridge_ports:
-  - eno3
-# TODO: we should have a bond on 25Gb, longer term
-# provision_oc_net_bond_slaves:
+  - bond0
+
+bond0_net_interface: bond0
+bond0_net_bond_slaves:
```
Comment (dougszumski) [inline etc/kayobe/inventory/group_vars/controllers/network-interfaces:16]: Do we need to set any of the bond options here, like bond mode for example?

https://kayobe.readthedocs.io/en/latest/configuration/network.html?highlight=bond

Same comment for other bonds.

Comment (dougszumski) [reply etc/kayobe/inventory/group_vars/controllers/network-interfaces:16]: Possibly some useful settings to be gleaned from here: https://github.com/VerneGlobal/kayobe-config/blame/master/etc/kayobe/inventory/group_vars/controllers/network-interfaces

Comment (JohnGarbutt) [reply etc/kayobe/inventory/group_vars/controllers/network-interfaces:16]: They are all set in networks.yaml

**Review by dougszumski — APPROVED** [review]

**Comment by JohnGarbutt** [comment]
Reworking this in a different branch, this one needs a rebase I think.

**Pull request closed without merging**